.github/workflows/k8s-content.yaml

```diff
@@ -0,0 +1,167 @@
+---
+name: Kubernetes content
+
+on:
+  push:
+    branches: [ 'master' ]
+
+env:
+  COSIGN_EXPERIMENTAL: 1
```

Comment:
Do we use this anywhere? It looks like we're specifying COSIGN_EXPERIMENTAL in each step explicitly.

Reply:
This is used by the cosign actions to dynamically take into use OIDC credentials, request a temporary key from fulcio, and sign. That way, we don't need to handle keys ourselves and pass those to sign the container and SBOM.